doc/recognition.xml

```diff
@@ -356,8 +356,17 @@ set this to <K>false</K>. The result is either <K>fail</K> or
 <K>false</K> if the method fails or a recognition info record <C>ri</C>.
 If the method created a leaf then <C>ri</C> will be a leaf, otherwise
 it will have the attribute <Ref Attr="Homom"/> set, but no factor or
-kernel have been created or recognised yet. You can use for example
-the methods in <Ref Var="FindHomMethodsPerm"/> or
+kernel have been created or recognised yet.
+
+<P/>GAP homomorphisms are not required to give a sensible answer when
```

Comment:
I like the idea behind this new section a lot! This is an important information that was missing in the documentation.

Though then this should be moved to another, possibly more discoverable, location. The GAPDoc comment you are adding this section to is for `TryFindHomMethod`. But I'm pretty sure `TryFindHomMethod` was intended as a debug function, namely as a way to apply a specific "FindHomMethod" to a group. It is never called anywhere.
Also see a source code comment in `gap/base/recognition.gi` added by @fingolfin in 2017:
```
# TODO: TryFindHomMethod is never called by anything
# in recog.
# Seems to be for user intervention and/or debugging?
```